eng/ci/host.coldstart.yml

```diff
@@ -0,0 +1,107 @@
+# No triggers for code push to any branch.
+trigger: none
+
+# No PR triggers for now
+pr: none
+
+schedules:
+  - cron: "0 1 * * *"
+    displayName: Nightly Build
+    branches:
+      include:
+      - dev
+    always: true
+
+resources:
+  repositories:
+  - repository: 1es
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+  - repository: eng
+    type: git
+    name: engineering
+    ref: refs/tags/release
+
+variables:
+  - template: /eng/ci/templates/variables/coldstart.yml@self
+  - template: /ci/variables/cfs.yml@eng
+
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc-benchmarking-large
+      image: 1es-windows-2022-benchmark-runner-vanilla
+      os: windows
+
+    stages:
+    - stage: RunWindows
+      displayName: Record latency(Windows)
+      jobs:
+      - ${{ each appId in split(variables.runInstances, ',') }}:
+        - template: /eng/ci/templates/official/jobs/run-coldstart.yml@self
```

Comment:
nit: I think if I recall correctly, yml files for unofficial pipeline will be under `templates/jobs` folder instead of `templates/official/jobs`

Reply:
what makes one pipeline official and one not?

Reply:
We are using Unofficial template in line 31 - `template: v1/1ES.Unofficial.PipelineTemplate.yml@1es`

Reply:
This pipeline is extending "1ES.Unofficial.PipelineTemplate" as this is considered as a non-prod pipeline. It appears that we have other pipelines which does the same thing (Extending "1ES.Unofficial.PipelineTemplate", but using jobs stored in  "official/jobs"). For example, this one: https://github.com/Azure/azure-functions-host/blob/dev/eng/ci/integration-tests.yml#L28-L38

We could reorganize them if needed. I am not opposed to that idea. Thoughts @jviau ?

Reply:
The `official` vs non-official differentiation is something I borrowed from how dotnet laid out their templates. The primary idea was the separate what is and isn't safe to run in `azfunc/public`. IE: if it needs service connections, it should go in `official`. Might be better to rename that folder `internal`.